normandy/classifier/admin/forms.py

```diff
@@ -0,0 +1,42 @@
+from collections import namedtuple
+
+from django import forms
+from django.contrib.admin import widgets
+from django.utils import timezone
+
+from normandy.recipes.models import Country, Locale, ReleaseChannel
+
+
+FakeClient = namedtuple('FakeClient', (
+    'locale',
+    'country',
+    'request_time',
+    'release_channel',
+    'user_id'
+))
```

Comment:
That we need this is pretty unsatisfying. I wish we could use the normal Client class for this, but that is complex. Maybe `Client` should be dumb and contain no logic, except in a `Client.from_request` constructor.

Out of scope for this PR though, I think.